core/src/thread_parker/windows/waitaddress.rs

```diff
@@ -47,6 +50,8 @@ impl WaitAddress {
         unsafe {
             // MSDN claims that that WaitOnAddress and WakeByAddressSingle are
             // located in kernel32.dll, but they are lying...
+
+            // REVIEW: how stable is this name to rely on it?
```

Comment:
It is fixed in the Windows ABI at this point. Apps would fail to launch if they removed this DLL.

Reply:
Can... that be backed up? Like point to some official documentation? Something like that? I can declare anything as stable as well but it doesn't mean much if it's wrong...